app/src/main/java/org/fossasia/openevent/dbutils/DbSingleton.java

```diff
@@ -90,6 +90,10 @@ public Version getVersionIds() {
         return databaseOperations.getVersionIds(mDb);
     }
 
+    public void addBookmarks(int id) {
+        databaseOperations.addToDb(id);
```

Comment:
What id? Can you be specific here? Can people book mark speakers, sessions, etc. Maybe you can allow bookmarking of anything? Just a thought.